nvflare/private/fed/server/fed_server.py

```diff
@@ -893,7 +893,7 @@ def deploy(self, args, grpc_args=None, secure_train=False):
                     prv_key_path=grpc_args["ssl_private_key"],
                 )
 
-        self.engine.cell = self.cell
+        self.engine.initialize_comm(self.cell)
```

Comment:
Probably better combine this initialize_comm(self.cell) with the following self._register_cellnet_cbs(). They both are doing the same for the cellnet callbacks set up.

Reply:
No, this part is for SP so that the ServerEngine will get the cell.  This is also where we really need to clean up our engine(s). It's very hard to follow.

Reply:
What I mean is these two lines could be combined. somehow HitHub comments shows the different highlight lines:

        self.engine.initialize_comm(self.cell)
        self._register_cellnet_cbs()